terraform/modules/cos/README.md

```diff
@@ -389,3 +389,20 @@ traefik:peers                            traefik:peers
 traefik:traefik-route                    grafana:ingress                          traefik_route            regular
 traefik:traefik-route                    tempo:ingress                            traefik_route            regular
 ```
+
+### Deploy COS on AWS EKS
+
+> **Note:** This option assumes that the required AWS infrastructure already exists and that a Juju controller has been bootstrapped.  
+> Additionally, a Juju model must be ready in advance.
+> 
+> See [provision AWS infrastructure](../aws-infra/README.md)
+
+1. Create a `aws.tfvars` file with the following content:
```

Comment:
Iirc, `cos` is intended to be used as a module, so we'd need a separate `main.tf` that imports the cos module, and uses `aws.tfvars`?